src/components/NcAppNavigationItem/NcAppNavigationItem.vue

```diff
@@ -175,8 +175,7 @@ Just set the `pinned` prop.
 				</div>
 				<NcActions v-show="displayActionsOnHoverFocus || forceDisplayActions"
 					ref="actions"
-					menu-align="right"
-					:container="'#' + id"
+					boundaries-element="#content-vue"
```

Comment:
If we set the container to `#app-navigation-vue`, the hidden overflow issue in #3146 is fixed as well, but additionally, the Actions menu is not jumping/moving when the body is scrolled.
```suggestion
					container="#app-navigation-vue"
					boundaries-element="#content-vue"
```